routes/movie.routes.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express");
+const router = express.Router();
+const middlewares = require("../helpers/middlewares");
+const controller = require("../controllers/movie.controller");
+
+// Routes
+module.exports = router;
+
+router.get("/", async (req, res) => {
+  controller
+    .getAllMovies()
+    .then((movies) => res.json(movies))
+    .catch((err) => res.json(err));
+});
+
+router.get("/:id", middlewares.mustBeInteger, async (req, res) => {
```

Comment:
I like how you integrated a declarative parameter validation using express middlware. Although IDs in most cases are represented as strings usually UUIDs and not an integer, and the reason is that the integer type would introduce a size limit which is impractical.